content/js/geolocation.md

```diff
@@ -1,68 +1,54 @@
-## Saving a location on record
+## Save a location on record
```

Comment:
I believe we will use a noun phrase for all these titles, i.e. "Saving" instead of "Save"